docs/general/builtins/scalar.rst

```diff
@@ -1780,6 +1780,16 @@ not matter.
 | ``d`` | only ``\n`` is considered a line-terminator when using ``^``, ``$`` |
 |       | and ``.``                                                           |
 +-------+---------------------------------------------------------------------+
+| ``g`` | do not stop at the first match, but rather keep going until the end |
+|       | of ``source``.                                                      |
++-------+---------------------------------------------------------------------+
```

Comment:
To match the language of the rest of the table, I might rephrase it as:

"do not stop at first match and keep going until end of ``source``. "